pkg/crc/machine/machine.go

```diff
@@ -136,6 +157,11 @@ func Start(startConfig StartConfig) (StartResult, error) {
 			return *result, errors.Newf("Error getting bundle metadata: %v", err)
 		}
 
+		if err := checkDiskImageSize(crcBundleMetadata); err != nil {
+			result.Error = err.Error()
+			return *result, errors.Newf("Error to match disk size %v", err)
```

Comment:
"Error checking disk image size"?
But it's not a great description of what the problem is, as it won't help in solving it. Maybe "Invalid bundle disk image '%s'", bundleinfo.GetDiskImagePath()?